Mada_Alwuthaynani/week_01/day_02/main.js

```diff
@@ -0,0 +1,54 @@
+var numOne=1;
+var numTwo=2;
+if(numOne>numTwo){
+    console.log(numOne);}
+    else if(numOne<numTwo){
+        console.log(numTwo);  
+    }
+else{console.log("equale");}
+var age=18;
+var wait=0;
+if(age<18){
+    wait=18-age;
+    console.log("Drive away! "+wait+" years to wait until you can drive"); }
+    else if(age>=18){console.log("Drive away!");}
+    
+    lang="Arabic";
+    if(lang=="English"){
+        console.log("Hello World");
+    }
+     else if(lang=="Arabic"){console.log("مرحبا بالعالم");}
+     else{console.log("french");}
+
+     var grade=80;
+     if(grade>=90){console.log("grade A")}
+      else if(grade>=80){console.log("grade B");}
+      else if(grade>=70){console.log("grade C");}
+      else if(grade>=60){console.log("grade D");}
+      else{
+        console.log("You Failed F");
+      }
+
+var airconditioner=true;
+var temperature=40;
+if(airconditioner && temperature>25){
+    console.log("Turn on the A/C Please");  
+}
+else if(!airconditioner && temperature>25){
+    console.log("Fix the A/C now! It's hot!");
+}
+else if(!airconditioner && temperature<25){
+    console.log("Fix the A/C whenever you have the chance... It's cool...");}
+var message="HEY";
+if(message.endsWith("?")){
+    console.log("sure");
+}
+else if(message===message.toUpperCase){
+    console.log("Woah, chill out!"); 
+}
+else if(message==" "||message==null){
+    console.log("Fine. Be that way!"); 
+}
+else{
+    console.log("Whatever");   
+}
```

Comment:
Nice work here!

Three things to note though:
1. `toUpperCase` is a function, which means we need to call it this way `.toUpperCase()`. You can read more about `toUpperCase()` [here](https://www.w3schools.com/jsref/jsref_toUpperCase.asp).
2. `message == " "` only checks the value of _message_ but doesn't check the type, which we care about in this scenario. So, if we do `message === " "` we make sure that the value inside _message_ is just _a space_ AND the type of _message_ is a _String_.
3. There are multiple ways to check if a String is _empty_; one way could be `message === ""`, however, `message === null` checks that the value inside of _message_ is an object of type `null`